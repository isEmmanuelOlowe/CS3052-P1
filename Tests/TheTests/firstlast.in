202
0.001 1.002
840485.369 396464.774
446583.435 353336.097
886428.433 318692.772
584090.22 15582.8494
383715.875 159368.627
58858.9136 691004.373
163545.951 899854.306
533064.714 159071.503
582699.021 604144.19
390478.195 269971.118
742377.406 293400.57
75538.0785 298525.606
857377.943 404982.634
662830.66 941968.323
2755.08143 846475.78
532596.024 462379.245
265612.235 787876.621
306785.131 982752.263
608715.653 600855.136
885895.131 212438.798
151859.864 304657.102
387476.951 337661.903
753553.276 643609.829
531628.252 603616.099
652488.447 459360.316
946370.486 327181.164
943890.339 368039.867
516599.95 7428.26172
24299.1556 272770.953
204963.51 591954.502
59368.6934 877693.35
302829.184 260842.552
498198.059 891495.22
286413.994 710025.581
675540.671 864923.577
959635.562 458489.973
376551.281 774675.406
354533.877 228639.116
669765.832 300318.248
565954.509 718966.572
390611.91 824465.313
844008.46 818766.311
943395.886 180467.77
520665.778 424886.765
913508.169 65643.7549
761364.127 882584.573
688256.842 398922.546
405008.799 761548.304
484633.905 125251.138
873121.166 222462.553
1413.962 528821.898
18697.3976 860513.813
242884.021 814891.942
965732.518 314571.844
809431.695 935560.105
220135.508 492109.153
289029.058 576353.738
261323.339 321066.976
1817.29961 173987.858
241174.551 44784.1404
701624.649 415451.188
503910.344 221845.691
393063.444 67030.2114
218142.211 479476.527
916203.193 219511.285
192693.937 350221.751
633681.628 211234.979
783411.413 53565.3932
444096.6 30673.9171
932180.218 176412.888
472844.835 909648.273
695567.358 871695.461
455090.556 930189.621
893304.001 398566.782
838734.383 693546.747
651269.549 739699.136
577212.317 678154.24
935388.054 273118.281
47412.5134 661938.885
618194.778 373038.39
377305.582 148627.986
25687.8246 644591.401
77018.8061 841380.513
255986.141 742633.917
377647.585 901844.23
211434.985 319529.925
251315.083 648532.5
250922.181 228644.317
136715.085 943221.162
548707.187 270060.767
865080.021 324021.447
680059.336 296703.348
876307.513 833147.254
73115.4782 649664.558
253581.256 898546.969
841889.925 611318.628
372587.091 832019.301
109214.906 757048.223
559329.326 850811.976
343096.982 857580.462
345197.295 691769.604
958707.883 893584.653
981544.357 121735.195
614776.279 55024.7948
376472.598 38126.6756
281923.955 525604.856
607242.966 560537.961
446534.988 816439.585
471854.014 27111.6801
292723.682 284898.305
17579.4171 195680.144
573165.442 829599.231
732976.481 104780.602
223888.398 119097.52
739051.619 947252.862
826131.565 821356.975
256495.729 250979.972
388221.673 338193.197
266409.127 527318.476
870894.566 401220.532
294573.819 45891.7929
560002.386 394246.563
822576.867 310821.141
91143.5897 475481.49
917044.094 262401.093
332093.346 978435.899
240634.874 902074.035
752286.033 373164.372
900627.876 457827.388
565923.483 422894.512
353870.876 970462.157
179091.359 431506.515
337176.765 215319.113
44725.7049 454367.753
62149.1404 683882.454
316171.237 551375.283
500765.995 268217.905
964826.539 62536.9632
806315.558 729575.791
705017.961 151974.411
842565.842 729957.446
955447.777 617928.583
527085.486 479388.126
607612.115 29386.5903
592769.648 47569.3664
904394.395 402646.93
310837.454 330131.069
734149.534 854944.744
340458.593 98626.5335
985167.064 401739.494
831962.391 629594.781
791226.69 437119.73
135134.9 59461.8604
930451.652 543823.899
389894.587 884899.019
856862.65 414557.516
107756.722 286758.584
777602.184 576675.995
808677.186 142250.931
410024.864 748094.222
344459.535 475115.767
23647.7474 416798.846
717891.538 525811.01
694865.075 236022.679
762750.408 779950.606
370517.959 121562.716
766006.338 928251.433
983660.708 289579.533
37813.0858 428106.916
15595.6066 598326.448
130679 756723.843
217496.166 374567.164
205663.394 577797.001
748369.189 787020.943
285268.327 455336.404
450007.519 850220.081
430246.027 835011.919
950872.11 330995.768
356794.046 248872.817
497053.138 417870.679
563165.348 885292.95
967759.368 685500.244
567805.379 182368.759
203961.873 29664.6427
547594.383 157764.928
211286.623 203990.633
193097.376 67294.043
782049.467 901521.526
422007.772 302589.854
797120.597 989525.426
181940.283 387471.607
737382.928 224731.394
664735.985 533507.071
468823.843 201830.975
97463.7872 566497.239
17311.6657 277023.692
886939.683 563884.072
62020.9302 70190.0742
307064.57 253306.766
134073.245 896639.856
0.002 0.999
