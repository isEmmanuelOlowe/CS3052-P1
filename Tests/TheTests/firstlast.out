0.00316227766
